packages/builder/src/steps/invoke.ts

```diff
@@ -107,7 +151,7 @@ async function runTxn(
     );
   }
 
-  if (config.fromCall && !contract.functions[config.fromCall.func]) {
+  if (config.fromCall && config.fromCall.func && !contract.functions[config.fromCall.func]) {
```

Comment:
Did you run into an issue with this? Seems like unrelated change

Reply:
I did, It was potentially coming back undefined since the fromCall parameter is optional, zod handles optional parameters as a union of ```type | undefined```